components/scg/src/scg-model-objects.js

```diff
@@ -60,6 +77,10 @@ SCg.ModelObject = function (options) {
     this.scene = null;
     this.bus = null;
     this.contour = null;
+    this.scaleElem = 1;
```

Comment:
please change number constants to something meaningful

Reply:
this is default value